test/e2e/pvutil.go

```diff
@@ -62,19 +62,6 @@ type persistentVolumeConfig struct {
 	namePrefix string
 }
 
-// Delete the nfs-server pod. Only done once per KubeDescription().
-func nfsServerPodCleanup(c clientset.Interface, config VolumeTestConfig) {
```

Comment:
Today I see `nfsServerPodCleanup` called from these 2 files:
    test/e2e/persistent_volumes.go
    test/e2e/persistent_volumes-disruptive.go
and defined in pvutil.go.
So, since it is called by 2 pv related files I still don't understand why it should not remain in pvutil.go...